pkg/crc/machine/bundle/metadata.go

```diff
@@ -275,7 +275,7 @@ func GetBundleInfoFromName(bundleName string) (*FilenameInfo, error) {
 		\.crcbundle              : Matches the file extension .crcbundle
 	*/
 	semverRegex := "(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)\\.(?:0|[1-9]\\d*)(?:-(?:(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+(?:[0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?"
-	bundleRegex := `crc(?:(?:_)([[:alpha:]]+))?_([[:alpha:]]+)_(%s)_([[:alnum:]]+)(?:(?:_)([0-9]+))?\.crcbundle`
+	bundleRegex := `crc(?:(?:_)([[:alpha:]]+))?_([[:alpha:]]+)_(%s)_([[:alnum:]]+)(?:(?:_)([0-9]+))?(?:(?:_)([0-9]+))?\.crcbundle`
```

Comment:
There’s a detailed comment before the regex, it needs to be updated.

Reply:
Instdea of duplicating `(?:(?:_)([0-9]+))?`, maybe the trailing `?` can be replaced by `{1,2)` or something like this?